src/service/internal_tables_access.h

```diff
@@ -822,6 +823,105 @@ namespace ccf
         {{uvm_endorsements->feed, {uvm_endorsements->svn}}});
     }
 
+    static void trust_static_snp_tcb_version(ccf::kv::Tx& tx)
+    {
+      auto h = tx.wo<ccf::SnpTcbVersionMap>(Tables::SNP_TCB_VERSIONS);
+
+      constexpr pal::snp::CPUID milan_chip_id{
+        .stepping = 0x1,
+        .base_model = 0x1,
+        .base_family = 0xF,
+        .reserved = 0,
+        .extended_model = 0x0,
+        .extended_family = 0x0A,
+        .reserved2 = 0};
+      constexpr pal::snp::TcbVersion milan_tcb_version = {
+        .boot_loader = 0,
+        .tee = 0,
+        .reserved = {0},
+        .snp = 0x18,
+        .microcode = 0xDB};
+      h->put(milan_chip_id.hex_str(), milan_tcb_version);
+
+      constexpr pal::snp::CPUID milan_x_chip_id{
+        .stepping = 0x2,
+        .base_model = 0x1,
+        .base_family = 0xF,
+        .reserved = 0,
+        .extended_model = 0x0,
+        .extended_family = 0x0A,
+        .reserved2 = 0};
+      constexpr pal::snp::TcbVersion milan_x_tcb_version = {
+        .boot_loader = 0,
+        .tee = 0,
+        .reserved = {0},
+        .snp = 0x18,
+        .microcode = 0x44};
+      h->put(milan_x_chip_id.hex_str(), milan_x_tcb_version);
+
+      constexpr pal::snp::CPUID genoa_chip_id{
+        .stepping = 0x1,
+        .base_model = 0x1,
+        .base_family = 0xF,
+        .reserved = 0,
+        .extended_model = 0x1,
+        .extended_family = 0x0A,
+        .reserved2 = 0};
+      constexpr pal::snp::TcbVersion genoa_tcb_version = {
+        .boot_loader = 0,
+        .tee = 0,
+        .reserved = {0},
+        .snp = 0x17,
+        .microcode = 0x54};
+      h->put(genoa_chip_id.hex_str(), genoa_tcb_version);
+
+      constexpr pal::snp::CPUID genoa_x_chip_id{
+        .stepping = 0x2,
+        .base_model = 0x1,
+        .base_family = 0xF,
+        .reserved = 0,
+        .extended_model = 0x1,
+        .extended_family = 0x0A,
+        .reserved2 = 0};
+      constexpr pal::snp::TcbVersion genoa_x_tcb_version = {
+        .boot_loader = 0,
+        .tee = 0,
+        .reserved = {0},
+        .snp = 0x17,
+        .microcode = 0x4F};
+      h->put(genoa_x_chip_id.hex_str(), genoa_x_tcb_version);
+    }
+
+    static void trust_node_snp_tcb_version(
+      ccf::kv::Tx& tx, pal::snp::Attestation& attestation)
+    {
+      // Fall back to statically configured tcb versions
+      auto cpuid = pal::snp::get_cpuid();
```

Comment:
I don't understand what this does, I don't think we should ever use values we receive from get_cpuid, because they are untrusted and open to HV manipulation, only values endorsed by attestation or members should ever be handled. What am I missing?